mail/dovecot/Portfile

```diff
@@ -254,152 +254,107 @@ if {${name} eq ${subport}} {
         description "Enable Percona (MySQL) support" {
             depends_lib-append          port:percona
             configure.env-append        MYSQL_CONFIG=${prefix}/lib/percona/bin/mysql_config
             configure.args-append       --with-mysql
     }
 
     variant mariadb \
-        conflicts mysql5 mysql51 mysql55 mysql56 percona \
+        conflicts mysql57 mysql8 mariadb10.1 percona \
         description "Enable MariaDB (MySQL) support" {
         depends_lib-append          port:mariadb
         configure.env-append        MYSQL_CONFIG=${prefix}/lib/mariadb/bin/mysql_config
         configure.args-append       --with-mysql
     }
 
-    variant mysql56 \
-        conflicts mysql5 mysql51 mysql55 mariadb percona \
-        description "Enable MySQL 5.6 support" {
-        depends_lib-append          port:mysql56
-        configure.env-append        MYSQL_CONFIG=${prefix}/lib/mysql56/bin/mysql_config
-        configure.args-append       --with-mysql
-    }
-
-    variant mysql55 \
-        conflicts mysql5 mysql51 mysql56 mariadb percona \
-        description "Enable MySQL 5.5 support" {
-        depends_lib-append          port:mysql55
-        configure.env-append        MYSQL_CONFIG=${prefix}/lib/mysql55/bin/mysql_config
+    variant mariadb10.1 \
+        conflicts mysql57 mysql8 mariadb percona \
+        description "Enable MariaDB 10.1 (MySQL) support" {
+        depends_lib-append          port:mariadb-10.1
+        configure.env-append        MYSQL_CONFIG=${prefix}/lib/mariadb-10.1/bin/mysql_config
         configure.args-append       --with-mysql
     }
 
-    variant mysql51 \
-        conflicts mysql5 mysql55 mysql56 mariadb percona \
-        description "Enable MySQL 5.1 support" {
-        depends_lib-append          port:mysql51
-        configure.env-append        MYSQL_CONFIG=${prefix}/lib/mysql51/bin/mysql_config
+    variant mysql57 \
+        conflicts mysql8 mariadb mariadb10.1 percona \
+        description "Enable MySQL 5.7 support" {
+        depends_lib-append          port:mysql57
+        configure.env-append        MYSQL_CONFIG=${prefix}/lib/mysql57/bin/mysql_config
         configure.args-append       --with-mysql
     }
 
-    variant mysql5 \
-        conflicts mysql51 mysql55 mysql56 mariadb percona \
-        description "Enable MySQL 5.1 support" {
-        depends_lib-append          port:mysql5
-        configure.env-append        MYSQL_CONFIG=${prefix}/lib/mysql5/bin/mysql_config
+    variant mysql8 \
+        conflicts mysql57 mariadb mariadb10.1 percona \
+        description "Enable MySQL 8 support" {
+        depends_lib-append          port:mysql8
+        configure.env-append        MYSQL_CONFIG=${prefix}/lib/mysql8/bin/mysql_config
         configure.args-append       --with-mysql
     }
 
-    variant no_startupitem description {Do not install a launchd plist} {}
+    variant startupitem_disable description {Do not install a launchd plist} {}
```

Comment:
This is still a negative variant. Just call it `startupitem` and do the opposite.

Reply:
Why is there even a variant for this? The user has to manually launch any startup item. I think this variant should be removed entirely.

Reply:
Thanks, done, with a little extra minor surgery.

Reply:
Oh, whoops, I did the surgery in https://github.com/macports/macports-ports/pull/6543.

I think it's a good idea to keep it there, unless you say otherwise.